src/sentry_plugins/jira/static/jira/components/issueActions.jsx

```diff
@@ -15,7 +15,7 @@ class IssueActions extends plugins.DefaultIssuePlugin.DefaultIssueActions {
     if (name === 'issuetype') {
       state.state = FormState.LOADING;
       this.setState(state, this.onLoad.bind(this, () => {
-        this.api.request(this.getPluginCreateEndpoint(), {
+        this.api.request(this.getPluginCreateEndpoint() + '?issue_type=' + value, {
```

Comment:
From my little understnaidng of this code, there's nothing that'd prevent `value` from being like, `&foo=1` or something else that'd break the url. So psure this would need to be uri encoded first.